### PR TITLE
Fix HTTPBody length calculation for utf8 strings.

### PR DESCRIPTION
**Pull request by tib**
This PR aims to fix the byte length calculation for utf8 strings.

### CODE REVIEW

**Review by czechboy0 — APPROVED** [review]
Great catch, thank you @tib!

**Comment by czechboy0** [comment]
Please rerun `./scripts/soundness` to fix up formatting and such.

File: Sources/OpenAPIRuntime/Conversion/ParameterStyles.swift [hunk 1]
```diff
@@ -45,6 +45,7 @@ extension ParameterStyle {
 
     /// Returns the default value of the explode field for the given style
     /// - Parameter style: The parameter style.
+    /// - Returns: Bool - True if the style is form, otherwise false
```
Comment (czechboy0) [inline Sources/OpenAPIRuntime/Conversion/ParameterStyles.swift:48]:
```suggestion
    /// - Returns: `true` if the style is form, otherwise `false`.
```

**Comment by czechboy0** [comment]
I'll do the 5.9 bump separately (https://github.com/apple/swift-openapi-runtime/pull/61), please back out any of the changes forced by swift-format 5.9 and use 5.8 instead.

File: Tests/OpenAPIRuntimeTests/Base/Test_OpenAPIValue.swift [hunk 2]
```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 import XCTest
-@_spi(Generated)@testable import OpenAPIRuntime
+@_spi(Generated) @testable import OpenAPIRuntime
```
[the remark this answers is not in the document]
Comment (tib) [reply Tests/OpenAPIRuntimeTests/Base/Test_OpenAPIValue.swift]: Reverted the changes, now I'm using swift-format for Swift 5.8.

**Review by czechboy0 — APPROVED** [review]

**Comment by czechboy0** [comment]
The soundness script is still unhappy it seems

**Comment by tib** [comment]
Now I've got this in Docker running 5.8, based on the dockerfile, previously I was using an Amazon Linux 5.8.1 instance and there were no soundness issues there... strange, tho. ¯\_(ツ)_/¯

**Pull request merged**